muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/guider/body/application/students/student.tsx

```diff
@@ -87,7 +87,9 @@ class Student extends React.Component<StudentProps, StudentState> {
       >
         <ApplicationListItemHeader>
           <span className="application-list__header-primary">
-            <span>{getName(this.props.student as any as UserType, true)}</span>
+            <span>
+              {getName(this.props.student as unknown as UserType, true)}
+            </span>
             <span className="application-list__header-helper">
```

Comment:
Why double type cast? Method getName first parameter takes any type parameter. If you want to indicate the actual type you are using, then UserType type cast enough.